### PR TITLE
Setup CI on new Jenkins

### PR DESCRIPTION
**Pull request by ehildenb**

### CODE REVIEW

**Review by SchmErik — APPROVED** [review]

File: kplutus [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+if [[ "$run_command" == 'version' ]] || [[ "$run_command" == '--version' ]]; then
+    notif 'KPlutus Version'
+    cat $INSTALL_LIB/version
+    exit 0
+fi
+
+backend="llvm"
+debug=false
+[[ ! "$run_command" == 'prove' ]] || backend='java'
+[[ ! "$run_command" =~ klab*   ]] || backend='java'
```
Comment (dwightguth) [inline kplutus:56]: this seems like it should be removed

File: kplutus [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+if [[ "$run_command" == 'version' ]] || [[ "$run_command" == '--version' ]]; then
+    notif 'KPlutus Version'
+    cat $INSTALL_LIB/version
+    exit 0
+fi
+
+backend="llvm"
+debug=false
+[[ ! "$run_command" == 'prove' ]] || backend='java'
```
Comment (dwightguth) [inline kplutus:55]: let's make this default to haskell

**Review by dwightguth — CHANGES_REQUESTED** [review]
some of the copy-paste needs to be audited, but overall it seems okay

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          reuseNode true
+        }
+      }
+      stages {
+        stage('Build') { steps { sh 'make build' } }
+        stage('Test') {
+          failFast true
+          options { timeout(time: 20, unit: 'MINUTES') }
+          parallel {
+            stage('Conformance') { steps { sh 'echo test-conformance' } }
```
[the remark this answers is not in the document]
Comment (ehildenb) [reply Jenkinsfile:28]: Yeah, it's just a placeholder.

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,151 @@
+# Settings
```
[the remark this answers is not in the document]
Comment (ehildenb) [reply Makefile:1]: The settings here are generic and will be needed for builds and installs, so would rather have it in place (same with kplutus script) to make reviews of changes to it more incremental.

But I stripped out a bunch of stuff that I missed in the first pass.

File: kplutus (hunk 1 above)
Comment (ehildenb) [reply kplutus]: Done

File: LICENSE [hunk 5]
```diff
@@ -1,67 +1,29 @@
-==============================================================================
-The Plutus Core Semantics in K Release License
-==============================================================================
-University of Illinois/NCSA
-Open Source License
+BSD 3-Clause License
 
-Copyright (c) 2017 University of Illinois at Urbana-Champaign.
+Copyright (c) 2010-2021, K Team
```
[the remark this answers is not in the document]
Comment (ehildenb) [reply LICENSE]: Done.

**Review by dwightguth — APPROVED** [review]

**Pull request merged**